src/Controller/ReviewController.php

```diff
@@ -4,84 +4,52 @@
 
+/**
+ * Review Controller
+ *
+ * @author  Pomian Ghe. Aurelian
+ */
 class ReviewController
 {
-	public function reviewAction(Application $app, Request $request, $id)
-	{
-		$form = $this->getReviewForm($app);
-
-		$companyRepository = new CompanyRepository($app);
-		$company = $companyRepository->findCompanyById($id);
-
-		if ($request->isMethod('POST')) {
-			$newReview = new Review();
-			$newReview->setFromArray($request->request->get($form->getName()));
-
-			$newReview->review_date = date('Y-m-d H:i:s');
-			$newReview->company_id = $id;
-
-			$form->handleRequest($request);
-			if($form->isValid())
-			{
-				$app['dbs']['mysql_read']->insert('review', $newReview->toArray());
-				return $app->redirect($app["url_generator"]->generate("company_details", ['id' => $id]));
-			}
-		}
-		return new Response($app['twig']->render('form/review_form.html.twig' ,[
-			'form' => $form->createView(),
-			'company' => $company
-		]));
-	}
-
-	protected function getReviewForm($app)
-	{
-        $form = $app['form.factory']->createBuilder(FormType::class)
-           	->add('name', TextType::class, array( 
-	            'label'  => ' ',
-	            'attr'   =>  array(
-	                'class'   => 'input-field'),
-	            'constraints' => new Assert\NotBlank()
-            ))
-            ->add('rating', ChoiceType::class, array (
-            	'choices' => array (
-            		'0.5' => 0.5,
-            		'1.0' => 1,
-            		'1.5' => 1.5,
-            		'2.0' => 2,
-            		'2.5' => 2.5,
-            		'3.0' => 3,
-            		'3.5' => 3.5,
-            		'4.0' => 4,
-            		'4.5' => 4.5,
-            		'5.0' => 5
-            	),
-            	'label' => ' ',
-            	'constraints' => new Assert\Length(array('min' => 0.5, 'max' => 5.0))
-            ))
-            ->add('comment', TextareaType::class, array(
-	            'label'  => ' ',
-	            'attr'   =>  array(
-	                'class'  => 'textarea-field'),
-	            'constraints' => new Assert\Length(['max' => 150])
-            ))
-            ->getForm();
-
-    	return $form;
-	}
-}
\ No newline at end of file
+    /**
+     * Handle review form page action and request.
+     * Route: /review/{id}
+     *
+     * @param  Application $app
+     * @param  Request     $request
+     * @param  int         $id
+     * @return Response
+     */
+    public function reviewAction(Application $app, Request $request, $id)
+    {
+        $reviewFormHelper = new ReviewFormHelper($app);
+        $form = $reviewFormHelper->getReviewForm($app);
+
+        $companyRepository = new CompanyRepository($app);
+        $company = $companyRepository->find($id);
+
+        if ($request->isMethod('POST')) {
+            $newReview = new Review();
+            $newReview->setFromArray($request->request->get($form->getName()));
+            $newReview->company_id = $id;
+
+            $form->handleRequest($request);
+            if($form->isValid())
```

Comment:
Please add a space after `if` and open `{` on the same line with the `if` statement.